Cocoapods Example/PortalSwift/Portal.swift

```diff
@@ -199,6 +203,54 @@ class PortalWrapper {
       print("Backup Status: ", status)
     }
   }
+    
+    func eject(backupMethod: BackupMethods.RawValue, user: UserResult, backupConfigs: BackupConfigs? = nil, completion: @escaping (Result<String>) -> Void) {
+        print("[PortalWrapper] Starting eject...")
+        // Setup request to get ciphert-text
+        let request = HttpRequest<CipherTextResult, [String: String]>(
+          url: CUSTODIAN_SERVER_URL! + "/mobile/\(user.exchangeUserId)/cipher-text/fetch",
+          method: "GET", body: [:],
+          headers: [:],
+          requestType: HttpRequestType.CustomRequest
+        )
+        
+
+        request.send { (result: Result<CipherTextResult>) in
+            guard result.error == nil else {
+                print("❌ [PortalWrapper] handleEject(): Error fetching cipherText:", result.error!)
+                return completion(Result(error: result.error!))
+            }
+
+            let cipherText = result.data!.cipherText
+            
+            // Setup request to get org-share
+            let requestOrgShare = HttpRequest<OrgShareResult, [String: String]>(
+              url: self.CUSTODIAN_SERVER_URL! + "/mobile/\(user.exchangeUserId)/org-share/fetch",
+              method: "GET", body: [:],
+              headers: [:],
+              requestType: HttpRequestType.CustomRequest
+            )
+            
+            requestOrgShare.send { (result: Result<OrgShareResult>) in
+                guard result.error == nil else {
+                    print("❌ [PortalWrapper] handleEject(): Error fetching orgShare:", result.error!)
+                    return completion(Result(error: result.error!))
+                }
+
+                let orgShare = result.data!.orgShare
```

Comment:
NIT: you could do if let orgshare = result.data.orgshare = {}
in order to not use the bang operator but its just the example app so no urgency

Reply:
if let orgShare = result.data.orgShare {
//all your code using orgShare 
} else { 
return 
}

Reply:
Added